portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -34,11 +34,18 @@ definition {
 
 		Navigator.openSiteURL(siteName = "Site Name");
 
-		Blogs.addEntry(
-			entryContent = "This is the blog test for AC",
-			entrySubtitle = "Blogs AC Subtitle",
-			entryTitle = "Blogs AC Title");
-
+		if (!(isSet(entryTitle))) {
+			Blogs.addEntry(
+				entryContent = "This is the blog test for AC",
+				entrySubtitle = "Blogs AC Subtitle",
+				entryTitle = "Blogs AC Title");
+		}
+		else {
+			Blogs.addEntry(
+				entryContent = "This is the blog test for AC",
+				entrySubtitle = "Blogs AC Subtitle",
+				entryTitle = "${entryTitle}");
+		}
```

Comment:
hi @joshitagaki , what do you think if we change the logic to this way in order to improve the readability:

```
		if (!(isSet(entryTitle))) {
			entryTitle = "Blogs AC Title"
		}

		Blogs.addEntry(
				entryContent = "This is the blog test for AC",
				entrySubtitle = "Blogs AC Subtitle",
				entryTitle = "${entryTitle}");
```

So, if the `entryTitle` is not set, we just set it with the default value. And then we read this value when calling `Blogs.addEntry` macro regardless if that value was set by our `isSet` check or if it had been already set by who called the macro.

Reply:
I was doing something similar at `ACLogin.loginAs()` and we can avoid the use of `else` clauses.